clinica/pipelines/dwi_preprocessing_using_fmap/dwi_preprocessing_using_phasediff_fmap_utils.py

```diff
@@ -152,7 +156,12 @@ def init_input_node(
 
     # Read metadata from DWI JSON file:
     [total_readout_time, phase_encoding_direction] = extract_metadata_from_json(
-        dwi_json, ["TotalReadoutTime", "PhaseEncodingDirection"]
+        dwi_json,
+        [
+            "TotalReadoutTime",
+            "PhaseEncodingDirection",
+        ],
+        handle_missing_keys_dwi=handle_missing_keys_dwi,
```

Comment:
The name of the parameter is `handle_missing_keys` and the value you pass is `handle_missing_keys_dwi`.

```suggestion
        handle_missing_keys=handle_missing_keys_dwi,
```